mycroft/listener/__init__.py

```diff
@@ -127,10 +127,19 @@ def __init__(self, loop):
         self.loop = loop
 
     @property
-    def wakeup_engines(self):
```

Comment:
Is this property intentionally removed and do we need to maintain a backwards-compat. reference?

Reply:
its unused dead code, but need to double check if present in mycroft-core, i believe its safe to remove

Reply:
Looks like this isn't in `mycroft-core` https://github.com/MycroftAI/mycroft-core/blob/dev/mycroft/client/speech/listener.py so should be safe to remove